src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +23,18 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        Function<String,List<Integer>> stringToIntList = s -> Arrays.stream(s.split(","))
```

Comment:
```suggestion
        return numbers.stream()
          .flatMap(n -> Arrays.stream(n.split(",")))
          .map(Integer::valueOf)
          ...
```
далі просто потрібно відфільтрувати числа які націло діляться на 2, викликати min(..).orElseThrow()
Тут не потрібно оголошувати додаткових змінних